Tests/UnitTests/ess-list-encs.component.spec.ts

```diff
@@ -0,0 +1,97 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
```

Comment:
Please add a test to prove `MaxEncLimit` config setting is honoured - i.e. add a test to prove the number of rows in the enc table does not exceed `MaxEncLimit`

Reply:
MaxEncLimit is used in upload component.
As Enc List table is already filtered using MaxEncLimit.
So that MaxEncLimit is not implemented in List Encs Component.

Reply:
Ok thanks Prasad

I notice the test '_**processEncFile should set raise "Some values have not been added to list." info**_' (in file Tests/UnitTests/ess-upload-file.component.spec.ts) has been removed in this PR

Are you saying it will be added back in a later PR?

Reply:
'processEncFile should set raise "Some values have not been added to list." info' test case was not required in upload file component
now it is added in ess-list-encs component
Please check ngOnInit of ess-list-encs component.

Reply:
So my understanding is unit test coverage for the "Some values have not been added to list" scenario has been removed by this PR.

And it will not be added back in this PR.

But unit test coverage will be added back by later work on the ess-list-encs.component

Is this correct?

Reply:
Test case "should set info message if displayErrorMessage is set to true on ngOnInit" already written in the ess-list-encs.spec file
Please check test case which is at line 51 in  ess-list-encs.spec file

Reply:
But all that is proving is ess-list-encs component displays an error message when essUploadFileService.infoMessage is true

It does not prove essUploadFileService.infoMessage is set to true when the number of encs exceeds the limit. I think this is (was) proved by the deleted test for ess-upload-file processEncFile()

Could you please reinstate the deleted test "Some values have not been added to list" and also improve it to prove essUploadFileService.infoMessage is set to true when the number of encs exceeds the limit